### PR TITLE
Upgrade to PostCSS v5

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by ai** [comment]
@TrySound do not include release commit, because sometimes maintainer want to other changes.

File: index.js [hunk 1]
```diff
@@ -50,16 +49,8 @@ function AtImport(options) {
...
-      styles &&
-      styles.nodes &&
-      styles.nodes[0] &&
-      styles.nodes[0].source &&
-      styles.nodes[0].source.input &&
-      styles.nodes[0].source.input.file
-    ) {
-      opts.from = styles.nodes[0].source.input.file
+    if (!opts.from && styles.source.input.file) {
+      opts.from = styles.source.input.file
```
Comment (MoOx) [inline index.js]: Not of the scope of the PR. Please revert this. Issues already happened and will happen again when people play with AST and introduce mistakes.

**Comment by MoOx** [comment]
Please remove the change that is not in the scope and squash in one commit.

**Comment by ai** [comment]
@MoOx ping

**Comment by MoOx** [comment]
@TrySound branch name was not accurate, something like "upgrade-postcss-5.x" would have been a better choice :)

**Comment by TrySound** [comment]
@MoOx Okay!

**Pull request merged**